include/glow/Graph/Graph.h

```diff
@@ -702,7 +702,8 @@ using VariableGradientsList = std::list<std::pair<Storage *, Storage *>>;
 /// \returns a new function with the name \p newFuncName.
 Function *differentiate(Function *F, const TrainingConfig &config,
                         llvm::StringRef newFuncName = "",
-                        VariableGradientsList *varGrads = nullptr);
+                        VariableGradientsList *varGrads = nullptr,
+                        Context *ctx = nullptr);
```

Comment:
I'm not sure about this way of doing this, but it was quick and didn't require me to change all the callsites.